src/utils/test.ts

```diff
@@ -131,3 +137,131 @@ export const verifyFolder = async (folder: Folder, parentFolderName = '.') => {
 
   return true
 }
+
+export const removeAllTargetsFromConfigs = async () => {
+  const { buildSourceFolder } = getConstants()
+  const configPath = path.join(buildSourceFolder, 'sasjs', 'sasjsconfig.json')
+  const config = await getConfiguration(configPath)
+  config.targets = []
+  await createFile(configPath, JSON.stringify(config, null, 1))
+
+  const globalConfig = await getGlobalRcFile()
+  if (globalConfig?.targets?.length) globalConfig.targets = []
+  await saveGlobalRcFile(JSON.stringify(globalConfig, null, 2))
+}
+
+export const updateTarget = async (target: Target, targetName: string) => {
+  const { buildSourceFolder } = getConstants()
+  const configPath = path.join(buildSourceFolder, 'sasjs', 'sasjsconfig.json')
+  const config = await getConfiguration(configPath)
+  if (config?.targets) {
+    const targetIndex = config.targets.findIndex((t) => t.name === targetName)
+    if (targetIndex >= 0) {
+      config.targets.splice(targetIndex, 1, {
+        ...config.targets[targetIndex],
+        ...target
+      })
+      await createFile(configPath, JSON.stringify(config, null, 1))
+    }
+  }
+}
+
+export const updateDocConfig = async (
+  docConfig: DocConfig,
+  target: string | undefined = undefined
+) => {
+  const { buildSourceFolder } = getConstants()
+  const configPath = path.join(buildSourceFolder, 'sasjs', 'sasjsconfig.json')
+  const config = await getConfiguration(configPath)
+  if (target) {
```

Comment:
Why is `target` a string, and why do we have this check?

Reply:
Previously, this function `updateDocConfig` is used to either update docConfig at root level or Target, for tests to check precedence.

Now this function is renamed to `updateConfig` as there's `updateTarget` as well, new commit is present.